magenta/models/music_vae/README.md

```diff
@@ -47,8 +47,18 @@ music_vae_train \
 --examples_path=/tmp/music_vae/mel_train_examples.tfrecord
 ```
 
-You should also launch an evaluation job (using ``--mode=eval`` with a heldout
-dataset) in order to compute metrics such as accuracy and to avoid overfitting.
+You will likely need to adjust some of the hyperparamters with the ``--hparams``
```

Comment:
shouldn't inline code be a single backtick?

https://guides.github.com/features/mastering-markdown/#syntax

Reply:
indeed, they both seem to work, but that saves us some storage costs :)